packages/page-bounties/src/useCounter.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017-2020 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { BountyIndex } from '@polkadot/types/interfaces';
+
+import { useEffect, useState } from 'react';
+
+import { useAccounts, useApi, useCall } from '@polkadot/react-hooks';
+
+export default function useCounter (): number {
+  const { hasAccounts } = useAccounts();
+  const { api, isApiReady } = useApi();
+  const bounties = useCall<BountyIndex>(isApiReady && hasAccounts && api.query.treasury.bountyCount);
```

Comment:
```suggestion
  const bounties = useCall<BountyIndex>(isApiReady && hasAccounts && api.query.treasury?.bountyCount);
```

Not all chains have treasury, or potentially not at the specific module.

Reply:
(What I did forget here, the `hasAccounts` is not needed, i.e. we can have the display without needing to have accounts created.